dev/main.ts

```diff
@@ -15,6 +22,54 @@ import 'bootstrap/dist/css/bootstrap.css';
 import 'bootstrap-vue/dist/bootstrap-vue.css';
 import 'bootstrap-vue/dist/bootstrap-vue-icons.min.css';
 
+class Invoice {
+  @DataGridField('String')
+  id!: string;
+
+  @DataGridField('String')
+  firstName!: string;
+
+  @DataGridField('String')
+  lastName!: string;
+
+  @DataGridField('String')
+  email!: string;
+
+  @DataGridField('Lang')
+  language!: string;
+
+  @DataGridField('Number')
+  note!: number;
+
+  @DataGridField('Money')
+  total!: number;
+
+  @DataGridField('InvoiceStatus')
+  status!: string;
+
+  @DataGridField('Date')
+  createdAt!: Date;
+}
+class Student {
```

Comment:
Where do you add `Student` and `Invoice`? Would it make sense to have an annotation `@DataGridEntity` to register the classes?